libraries/AP_Motors/AP_MotorsMatrix.cpp

```diff
@@ -560,6 +560,45 @@ void AP_MotorsMatrix::remove_motor(int8_t motor_num)
     }
 }
 
+void AP_MotorsMatrix::add_motors(const struct MotorDefInt *motors, uint8_t num_motors)
+{
+    for (uint8_t i=0; i<num_motors; i++) {
+        const auto &motor = motors[i];
+        add_motor(motor.motor_num, motor.angle_degrees, motor.yaw_factor, motor.testing_order);
+    }
+}
+
+static const AP_MotorsMatrix::MotorDefInt motors_quad_plus[] {
+    { AP_MOTORS_MOT_1,   90, AP_MOTORS_MATRIX_YAW_FACTOR_CCW,  2 },
```

Comment:
We could loose this first number too and just use the array index.

Reply:
Yes, I believe we can.  I decided not to in this PR in case anybody had an emotional attachment to it.  And I spotted the strict-sequencing late :-)